src/derive/incflo_error.cpp

```diff
@@ -71,8 +75,6 @@ void incflo::DiffFromExact (int /*lev*/, Geometry& lev_geom, Real time, Real dt,
             constexpr Real u0 = Real(1.0);
             constexpr Real v0 = Real(1.0);
 
-            constexpr Real visc_coef = Real(0.001);
```

Comment:
@michaeljbrazell Couldn't we do `Real visc_coef = m_mu` instead of parsing again?

Reply:
That's kind of a style question   The parsing is essentially free though, since the database is read all at once and stuffed away somewhere.

Reply:
yeah I guess either would work. I didn't know that m_mu was in the incflo.H